### PR TITLE
Changing some style issues for lectures

### PR DESCRIPTION
**Pull request by Problemkerze**
possible solution for issue #3033

### CODE REVIEW

File: indico/htdocs/sass/themes/indico.scss [hunk 1]
```diff
@@ -577,6 +576,7 @@ div.event-details {
 
     .event-details-content {
         flex: 1;
+		padding-right:20%;
```
Comment (ThiefMaster) [inline indico/htdocs/sass/themes/indico.scss]: add a space after `:`, and indent using spaces, not tabs

File: indico/htdocs/sass/themes/indico.scss [hunk 2]
```diff
@@ -231,7 +231,7 @@ div.event-header {
     &.subcontrib-description {
         font-size: 0.9em;
         padding: 0.5em 0;
-    }
+		}
```
Comment (ThiefMaster) [inline indico/htdocs/sass/themes/indico.scss]: this change shouldn't be here

File: indico/htdocs/sass/themes/indico.scss [hunk 3]
```diff
@@ -577,6 +576,7 @@ div.event-details {
 
     .event-details-content {
         flex: 1;
+        padding-right:20%;
```
Comment (ThiefMaster) [inline indico/htdocs/sass/themes/indico.scss]: missing space after `:`

File: indico/htdocs/sass/themes/indico.scss [hunk 4]
```diff
@@ -560,7 +560,6 @@ div.event-details {
 
     .event-details-row {
         display: flex;
-        align-items: baseline;
```
Comment (nop33) [inline indico/htdocs/sass/themes/indico.scss]: I don't see how this fixes anything. In my opinion, it brakes the alignment. To demonstrate:

With `align-items: baseline` the title is aligned in the same row as the content:
image


With your fix:
image

**Comment by nop33** [comment]
This PR doesn't add anything new, nor fixes any error. @Problemkerze if you find any problems, don't hesitate to open a new issue/PR.

**Comment by Problemkerze** [comment]
@nop33: As you can see in issue #3033, with `align-items: baseline` in some cases the text is not aligned as it should.
Changes without baseline can be seen here:
baseline

**Comment by pferreir** [comment]
Usual problem: I think `base-align` looks great when there's text, not so nice if there are images. I can't see an easy way to solve that.
Maybe the best would be to actually remove the "Description" label and find a way to make it clear this is the description? I never liked those labels.

**Pull request closed without merging**